tests/firebase-tests/firebase-access-tests.jsx

```diff
@@ -179,7 +179,50 @@ describe("test advanceDay", () => {
   }).timeout(0);
 });
 
-// TODO: test getLeaders
+var USER2ID;
+
```

Comment:
a general rule of unit test is to test behavior instead of functions. e.g. " test add user successfully" "test get chart urls failed".....so it does have to be exactly one function to one test. Just some thoughts :)

Reply:
ah, I see. so should each "describe" have only one test?

Reply:
not necessary. describe should group tests that have a similar behavior. The tests look good. It's just i saw your test description is about "test + method name", maybe you can just update it to make it clearer

Reply:
gotcha, will fix that up!